dask_ml/datasets.py

```diff
@@ -82,6 +82,48 @@ def make_counts(n_samples=1000, n_features=100, n_informative=2, scale=1.0,
     return X, y
 
 
+def make_poisson(n_samples=1000, n_features=100, n_informative=2, scale=1.0,
```

Comment:
This seems to be exactly the same as `make_counts` above?

Reply:
Whoops, thanks.